tensorflow/core/common_runtime/gpu/gpu_device.cc

```diff
@@ -93,7 +93,7 @@ using se::cuda::ScopedActivateExecutorContext;
 
 typedef hipStream_t gpuStream_t;
```

Comment:
`#if GOOGLE_CUDA` section above now becomes incorrect. In fact now we probably could merge these 2 sections together.

Reply:
the `#define EIGEN_GPU_SCRATCH_SIZE` is now the same for both `GOOGLE_CUDA` and `TENSORFLOW_USE_ROCM`, rest still need to be different